code/modules/mob/mob.dm

```diff
@@ -674,6 +674,11 @@ var/next_mob_id = 0
 //Updates canmove, lying and icons. Could perhaps do with a rename but I can't think of anything to describe it.
 //Robots, animals and brains have their own version so don't worry about them
 /mob/proc/update_canmove()
+	var/crit = 0
+	if(isliving(src))
```

Comment:
why not put in mob/living/proc/update_canmove() along with ..()

More importantly why is there a hook here to trigger crit